lib/stinger_core/src/stinger.c

```diff
@@ -1410,6 +1465,8 @@ stinger_remove_edge_pair (struct stinger *G,
 
 MTA("mta parallel default")
 
+/* FIXME: Doesn't follow the STINGER direction standards.  Only inserts out edges */
```

Comment:
This will put in the in edges if they are provided.  This comment can be removed since we got rid of the standalone directory